README.md

```diff
@@ -250,6 +251,119 @@ class UsersController < ApplicationController
 end
 ```
 
+### Custom input errors
+
+Use a `Hash` with `is:` and `message:` keys to prepare custom
+error messages on inputs. For example:
+
+```rb
+class UpdateAdminUser < Actor
+  input :user,
+        must: {
+          be_an_admin: {
+            is: -> user { user.admin? },
+            message: "The user is not an administrator"
+          }
+        }
+
+  # ...
+end
+```
+
+You can also use incoming arguments when shaping your error text:
+
+```rb
+class UpdateUser < Actor
+  input :user,
+        allow_nil: {
+          is: false,
+          message: (lambda do |origin:, input_key:, actor:|
```

Comment:
What do you think about using a hash here for arguments, instead?
See: https://github.com/sunny/actor/pull/82#discussion_r967784006

Reply:
Oops, I forgot about that comment. I like it, maybe at the moment this is a really good way out of a small problem.

Reply:
Yeah, in the library this wouldn’t change anything but I think it would be good in the documentation to encourage either the hash approach or the `**` approach so that when adding a new arguments we don’t break existing apps.

Reply:
Okay, got it. I will update the README and add `**`.